client/src/modules/stock/entry/entry.html

```diff
@@ -95,8 +115,8 @@
     </div>
 
     <!-- grid -->
-    <div id="stock-entry-grid" 
-      ui-grid="StockCtrl.gridOptions" 
+    <div id="stock-entry-grid"
+      ui-grid="StockCtrl.gridOptions"
       style="height: 300px; width: 100%;"
```

Comment:
It would be nice if these styles were contained in the `bhima-bootstrap.less` file.  This should probably be a separate PR that addresses it here and in all other modules that have this style of `ui-grid`.